dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/src/ResourceContext.cpp

```diff
@@ -101,6 +103,16 @@ void ResourceContext::Apply()
             winrt::check_hresult(MrmSetQualifier(m_resourceContext, eachValue.Key().c_str(), eachValue.Value().c_str()));
         }
     }
+
+    // sync with Windows::Globalization::ApplicationLanguages::PrimaryLanguageOverride if it has been updated more recently
+    if (AppModel::Identity::IsPackagedProcess())
+    {
+        auto language = winrt::Windows::Globalization::ApplicationLanguages::PrimaryLanguageOverride();
```

Comment:
do we need to check this every time loading resource? can we do it once in initialization, and if it's set, make MWGA.PrimaryLanguageOverride not settable?

Reply:
I think the primary obstacle with that approach is one of timing... unless the developer explicitly listens to the `Application.ResourceManagerRequested` event then WinUI is going to use MRT Core before custom user code runs. This would then result in WGA.PrimaryLanguageOverride (seemingly) never being respected.

Ideally there would be an event on WGA that we could listen to for changes to PrimaryLanguageOverride (and similarly for MWGA) but in the absence of such an event this seems to me the only way to ensure correct behavior.

Reply:
https://learn.microsoft.com/en-us/uwp/api/windows.applicationmodel.resources.core.resourcequalifierobservablemap.mapchanged?view=winrt-26100 listens for language change, but we probably don't want a dependency on that. Or the override is essentially stored in a registry. Probably out of scope though.
MWGA.PrimaryLanguageOverride is persisted. so restarting app would solve that timing issue. not sure whether that's acceptable.
just some thoughts. i am ok with leaving it alone. Hopefully reading is fast enough.

Reply:
Well, developers _don't_ like restarting their apps in order for language changes to propagate. :)

We can always revisit if benchmarks/anecdata indicate that this is causing perf issues.